build_tools/travis/before_install.sh

```diff
@@ -28,6 +28,10 @@ if [[ $TRAVIS_OS_NAME == 'osx' ]]; then
         brew install vtk
     fi
 
+    if [[ "$USE_OMP" == 1 ]]; then
+        brew install llvm
+    fi
+
```

Comment:
I have no idea.

Before clang we had gcc 4.2 on mac and I did the previous release using the intel compiler anyway...